adaptive_components/lib/src/adaptive_container.dart

```diff
@@ -61,7 +63,19 @@ class AdaptiveContainer extends StatelessWidget {
   ///
   /// This is used by the builder to see what type of screen the user wants this
   /// [AdaptiveContainer] to fit within.
-  final AdaptiveConstraints adaptiveConstraints;
+  AdaptiveConstraints adaptiveConstraints;
+
+  /// AdaptiveColumn is used with [AdaptiveColumn] and [AdaptiveRow] to represent
+  /// the amount of columns that this widget will fill up within a certain [Flex]
+  /// range.
+  ///
+  /// By default the columns will only represent one column space. If you want
+  /// this content of this widget to be shown increase it. There can be at most
+  /// 12 columns per flex range.
+  ///
+  /// Learn more by visiting the Material website:
+  /// https://material.io/design/layout/responsive-layout-grid.html#breakpoints
+  final int adaptiveColumn;
```

Comment:
I suggest renaming as this is a bit confusing. Does it represent the number of columns?

Reply:
Yeah it does. This woud represent the amount of column space it would take up within the range of the screen. If it is an xs screen and the adaptiveColumn is set to 4 then the container would take up the entire screen